src/models/events/push.rs

```diff
@@ -178,3 +186,77 @@ pub struct SlackLinkObject {
     pub domain: String,
     pub url: Url,
 }
+
+#[skip_serializing_none]
+#[derive(Debug, PartialEq, Clone, Serialize, Deserialize, Builder)]
+pub struct MemberJoinedChannelEvent {
+    pub user: SlackUserId,
+    pub channel: SlackChannelId,
+    pub channel_type: SlackChannelType,
+    pub team: SlackTeamId,
+    pub inviter: Option<SlackUserId>,
+}
+
+#[skip_serializing_none]
+#[derive(Debug, PartialEq, Clone, Serialize, Deserialize, Builder)]
+pub struct MemberLeftChannelEvent {
+    pub user: SlackUserId,
+    pub channel: SlackChannelId,
+    pub channel_type: SlackChannelType,
+    pub team: SlackTeamId,
+}
+
+#[skip_serializing_none]
+#[derive(Debug, PartialEq, Clone, Serialize, Deserialize, Builder)]
+pub struct ChannelCreatedEvent {
+    pub channel: ChannelCreatedObject,
```

Comment:
I think you can use here `SlackChannelInfo` instead of creating a new type: `ChannelCreatedObject`.

Reply:
It seems like that object for example does not have `name` instead it has `name_normalized`? Or is name auto mapped to name_normalized?

Reply:
Hm, that model seems missing `name` field:
https://api.slack.com/methods/conversations.info
So we can add that field there 👍🏻

Reply:
In this particular case, I think it is really the same, and unfortunately Slack has a lot of optional fields and ambiguous types, so I won't be surprised if they return both or one of them for some cases. I wouldn't want to fix multiple types for that.

Reply:
Ok I switched both channel created and channel renamed to use channel info object and added `name` to the channel info object